src/usr/local/www/system_advanced_admin.php

```diff
@@ -252,15 +259,24 @@
 			unset($config['system']['ssh']['sshdkeyonly']);
 		}
 
+		$sshd_agentforwarding = isset($config['system']['ssh']['sshdagentforwarding']);
+		if ($_POST['sshdagentforwarding']) {
+			$config['system']['ssh']['sshdagentforwarding'] = true;
+		} else {
+			unset($config['system']['ssh']['sshdagentforwarding']);
+		}
+
 		$sshd_port = $config['system']['ssh']['port'];
 		if ($_POST['sshport']) {
 			$config['system']['ssh']['port'] = $_POST['sshport'];
 		} else if (isset($config['system']['ssh']['port'])) {
 			unset($config['system']['ssh']['port']);
 		}
 
+
```

Comment:
There is an extraneous blank line here

Reply:
Not really because this separates the new block from next one, check the suroundings and you will see that the newline is desirable (github diff can confuse here).

Reply:
There was already one blank line, this adds another one